### PR TITLE
Benchmark Changes

### PR DESCRIPTION
**Pull request by kanav99**
Ref: https://github.com/JuliaDiffEq/OrdinaryDiffEq.jl/pull/922#issuecomment-529155144

@devmotion @ChrisRackauckas please check the tolerances and changes in the Orego script.

### CODE REVIEW

File: diagrams/orego.jl [hunk 1]
```diff
@@ -16,39 +15,78 @@ test_sol = TestSolution(sol)
...
 
 wp = WorkPrecisionSet(prob,abstols,reltols,setups;verbose=false,
                       dense=false,appxsol=test_sol,maxiters=Int(1e5),error_estimate=:l2)
-DIAGRAMS["OregoLowTol2"] = plot(wp)
+DIAGRAMS["Orego-LowTol-Group1-2"] = plot(wp)
 
 wp = WorkPrecisionSet(prob,abstols,reltols,setups;
                       appxsol=test_sol,maxiters=Int(1e5),error_estimate=:L2)
-DIAGRAMS["OregoLowTol3"] = plot(wp)
+DIAGRAMS["Orego-LowTol-Group1-3"] = plot(wp)
```
Comment (ChrisRackauckas) [inline diagrams/orego.jl]: we're going to need more than just OREGO to determine regressions

Comment (kanav99) [reply diagrams/orego.jl]: Yeah they are going to be almost same, I just edited this to get a review

File: diagrams/orego.jl [hunk 2]
```diff
@@ -16,39 +15,78 @@ test_sol = TestSolution(sol)
...
+          Dict(:alg=>RadauIIA5()),
+          Dict(:alg=>CVODE_BDF()),
+          Dict(:alg=>lsoda())]
+wp = WorkPrecisionSet(prob,abstols,reltols,setups;
+                      save_everystep=false,appxsol=test_sol,maxiters=Int(1e5))
+DIAGRAMS["Orego-HighTol-Group2-1"] = plot(wp)
+
+
+wp = WorkPrecisionSet(prob,abstols,reltols,setups;dense = false,verbose=false,
+                      appxsol=test_sol,maxiters=Int(1e5),error_estimate=:l2)
```
Comment (ChrisRackauckas) [inline diagrams/orego.jl]: just :l2 should be fine.

File: diagrams/orego.jl [hunk 3]
```diff
@@ -16,39 +15,78 @@ test_sol = TestSolution(sol)
...
+DIAGRAMS["Orego-LowTol-Group1-3"] = plot(wp)
+
+
+# group 2
+
+# High Tolerance
+abstols = 1 ./ 10 .^ (5:8)
+reltols = 1 ./ 10 .^ (1:4)
+
+setups = [Dict(:alg=>Rodas4()),
```
Comment (ChrisRackauckas) [inline diagrams/orego.jl]: these ones should only be low tolerance.

File: diagrams/orego.jl [hunk 4]
```diff
@@ -16,39 +15,78 @@ test_sol = TestSolution(sol)
 abstols = 1 ./ 10 .^ (5:8)
 reltols = 1 ./ 10 .^ (1:4);
 setups = [Dict(:alg=>Rosenbrock23()),
-          Dict(:alg=>Rodas3()),
-          Dict(:alg=>TRBDF2())]
+          Dict(:alg=>TRBDF2()),
```
Comment (ChrisRackauckas) [inline diagrams/orego.jl]: these ones should only be high tolerance.

**Pull request merged**